web/react/utils/utils.jsx

```diff
@@ -753,8 +753,23 @@ module.exports.isValidUsername = function(name) {
     return error;
 };
 
+function checkRTL(string) {
```

Comment:
Lets chat.  I think there is a problem with the original bug description.  We should use the browsers encoding to detect RHL and display appropriately.